doc/pre-receive.sample

```diff
@@ -9,7 +9,10 @@
 # either set an environment variable machine wide `GITGUARDIAN_API_KEY`
 # or uncomment and insert your key in the following line
 
-#export GITGUARDIAN_API_KEY=<INSERT YOUR KEY>
+# export GITGUARDIAN_API_KEY=<INSERT YOUR KEY>
+
+# (set this if you are using a self-hosted GitGuardian instance)
+# export GITGUARDIAN_API_URL="https://dashboard.gitguardian.mycorp.local/exposed/"
```

Comment:
Is the "exposed/" part always there for self-hosted instances? If not I would use a simpler URL for the example.

Reply:
Yes, after checking the `exposed` part is necessary for self-hosted

Reply:
OK, all good then :+1: